osv/ecosystems/rocky_linux.py

```diff
@@ -0,0 +1,36 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Rocky linux ecosystem helper."""
+
+from ..third_party.univers.rpm import RpmVersion
+from .helper_base import Ecosystem
+
+
+class RockyLinux(Ecosystem):
+  """"Rocky Linux ecosystem"""
+
+  @property
+  def name(self):
+    return "Rocky Linux"
```

Comment:
Is this `name` still necessary?

Reply:
Not for this PR, but I think it would be better to keep it as the ecosystem name does not exactly match the class name.